Netatmo improving Battery info

Jypy opened the pull request:
**Description:**


**Related issue (if applicable):** fixes #<home-assistant issue number goes here>

**Pull request in [home-assistant.github.io](https://github.com/home-assistant/home-assistant.github.io) with documentation (if applicable):** home-assistant/home-assistant.github.io#<home-assistant.github.io PR number goes here>

**Example entry for `configuration.yaml` (if applicable):**
```yaml

```

**Checklist:**

If user exposed functionality or configuration variables are added/changed:
  - [x] Documentation added/updated in [home-assistant.github.io](https://github.com/home-assistant/home-assistant.github.io)

If the code communicates with devices, web services, or third-party tools:
  - [x] Local tests with `tox` run successfully. **Your PR cannot be merged unless tests pass**
  - [x] New dependencies have been added to the `REQUIREMENTS` variable ([example][ex-requir]).
  - [x] New dependencies are only imported inside functions that use them ([example][ex-import]).
  - [x] New dependencies have been added to `requirements_all.txt` by running `script/gen_requirements_all.py`.
  - [x] New files were added to `.coveragerc`.

If the code does not interact with devices:
  - [x] Local tests with `tox` run successfully. **Your PR cannot be merged unless tests pass**
  - [x] Tests have been added to verify that the new code works.

[ex-requir]: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/keyboard.py#L16
[ex-import]: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/keyboard.py#L51

Improving battery status info (to reflect NetAtmo API documentation and change deprecated DeviceList for WeatherStationData

Jypy committed:
Improving Battery info

Improving battery status info (to reflect NetAtmo API documentation and change deprecated DeviceList for WeatherStationData

jabesq commented:
Looks good to me

fabaff commented:
For me this looks like the perfect task for a template sensor: Converting numerical values into a textual representation. My guess is that some people would prefer to have a graph instead of the output text.

Just the 0.02 cents from a non-Netatmo user...

balloob commented:
I agree with @fabaff, could you also add the voltage as a number to the attributes in case someone wants to plot that data?

You'll also need to address the hound issue.

Jypy committed:
Fixes from previous update

Fix the hound issue.
add battery_lvl, WindAngle_value, GustAngle_value, rf_status_lvl, wifi_status_lvl

Jypy commented:
I've submitted a new version implementing what you suggested and a fix for the long line issue.

balloob commented:
Looks good! :dolphin:

Jypy commented:
Thanks! Documentation updated :)
